devices/tuya.js

```diff
@@ -849,6 +849,20 @@ module.exports = [
         toZigbee: [],
         exposes: [e.occupancy(), e.battery_low(), e.tamper(), e.battery()],
     },
+    {
+        fingerprint: [{modelID: 'TS0202', manufacturerName: '_TZ3000_lltemgsf'}],
+        model: 'TS0202_2',
```

Comment:
This can be merged with `TS0202_1`

Reply:
ok, but _TZ3000_lltemgsf dosn't have battery and tamper
only occupancy, battery_low

Reply:
@groenmarsmannetje does the `tamper` work for your TS0202_1 motion sensor?

Reply:
@koenkk I do not think so. Will double check

Reply:
I checked and the tamper sensor doesn’t turn on when I open the device to remove battery.

Reply:
Thanks!